### PR TITLE
Retain input tags order on resizing

### LINKED ISSUE #515: Elements of password tab get disarranged after resizing
**Actual Behaviour**
Elements of password tab get disarranged after resizing.

**Expected Behaviour**
Elements should remain in a proper order.

**Steps to reproduce it**
Resize the password tab.

**Screenshots of the issue**

screen shot 2018-09-21 at 12 56 00 am

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by neeraj3029**
Fixes #515 

Changes: 
Elements of password tab remain in proper order even after resizing

Surge Deployment Link: https://pr-518-fossasia-susi-accounts.surge.sh

Screenshots for the change:

screen shot 2018-09-21 at 12 05 25 pm

resize2

### CODE REVIEW

File: src/components/Auth/ChangePassword/ChangePassword.css [hunk 1]
```diff
@@ -29,3 +28,13 @@
 .forgot:hover {
   text-decoration: underline;
 }
+
+.submitbtn{
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.css]: `. prop {`

File: src/components/Auth/ChangePassword/ChangePassword.css [hunk 2]
```diff
@@ -29,3 +28,13 @@
...
+  float: left;
+  max-width: 300px;
+  margin: 0 auto;
+}
+
+.label {
+  width: 150px;
+}
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.css]: EOF

File: src/components/Auth/ChangePassword/ChangePassword.css [hunk 3]
```diff
@@ -29,3 +28,13 @@
...
   text-decoration: underline;
 }
+
+.submitbtn{
+  float: left;
+  max-width: 300px;
+  margin: 0 auto;
+}
+
+.label {
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.css]: This additional property is not needed when you already have a style object for labels

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 4]
```diff
@@ -217,7 +217,9 @@ export default class ChangePassword extends Component {
         <div className="changePasswordForm">
           <Paper zDepth={0} style={styles}>
             <form onSubmit={this.handleSubmit}>
-              <div style={labelStyle}>Current Password</div>
+              <div className="label" style={labelStyle}>
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: Don't use className css when there's already a style prop

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 5]
```diff
@@ -233,7 +235,9 @@ export default class ChangePassword extends Component {
                 />
               </div>
               <br />
-              <div style={labelStyle}>New Password</div>
+              <div className="label" style={labelStyle}>
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: same

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 6]
```diff
@@ -250,7 +254,9 @@ export default class ChangePassword extends Component {
                 />
               </div>
               <br />
-              <div style={labelStyle}>Verify Password</div>
+              <div className="label" style={labelStyle}>
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: same

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 7]
```diff
@@ -268,19 +274,22 @@ export default class ChangePassword extends Component {
...
               <br />
-              <div>
-                <RaisedButton
-                  label="Save New Password"
-                  type="submit"
-                  style={{ marginLeft: '30%', marginTop: '0px' }}
-                  backgroundColor={ChatConstants.standardBlue}
-                  labelColor="#fff"
-                />
+              <div className="submitbtn">
```
Comment (anshumanv) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: I'd advice you to use some other style object like used for label

**Comment by neeraj3029** [comment]
@anshumanv Please take a look.

**Comment by anshumanv** [comment]
Seems good, waiting for travis to pass.

**Comment by neeraj3029** [comment]
I'll take care of the commit messages and make the required changes. 😅

**Comment by neeraj3029** [comment]
@stealthanthrax the present commit message is the changed one.

**Comment by anshumanv** [comment]
Looks good I think, merging :rocket:

**Pull request merged**